File: 00_App_Java/app/src/main/java/com/example/a00_app_java/MainActivity.java
package com.example.a00_app_java;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);

        Button boton1  = findViewById(R.id.boton1);

        boton1.setOnClickListener(view -> {
            Log.i("MainActivity" , "Boton Pulsado! ");
        });

        Button boton2 = findViewById(R.id.boton2);

        boton2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Log.i("MainActivity", "BOTON 2 PULSADO!");
            }
        });
    }
}

File: 00_AppKotlin/app/src/main/java/com/example/a00_appkotlin/MainActivity.kt
package com.example.a00_appkotlin

import android.os.Bundle
import android.util.Log
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

//Para extender de las clases en Kotlin se pone ':'
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        var user = Usuario()
        user.nombre = "Piero"
        Log.d("MainActivity" , "Nombre:" + user.nombre)


        val boton1 = findViewById<Button>(R.id.boton1)
        val boton2 : Button = findViewById(R.id.boton2)

        boton1.setOnClickListener ({
          Log.d("MainActivity", "Has pulsado el boton 1")
        })

        boton2.setOnClickListener{ v ->
            Log.d("MainActivy", "Pulsando el boton 2 " + v.toString())
            Log.d("MainActivy", "Esto es otra linea")
        }









        }
    }